src/data/index_array_type.js

```diff
@@ -1,12 +1,8 @@
 // @flow
 
-/**
```

Comment:
And this one. Was their removal a codemod effect? If so, please sweep for all cases.

Reply:
Ack, yes -- I caught this happening with the `// @flow` comments and fixed it, but didn't realize it was affecting subsequent comments as well.  Will fix.